coreos/fedora-coreos-config
# overlay.d & tests: Add alternatives migration and test and remove iptables-legacy

overlay.d: Add empty statoverride config files

---

overlay.d & tests: Add alternatives migration and test

- Add an overlay with the migration logic for alternatives
- Add a test for the migration script

This should make sure that the system is setup properly and that the
migration script will do the right thing on older systems.

See: https://github.com/coreos/fedora-coreos-tracker/issues/1818
See: https://github.com/coreos/fedora-coreos-tracker/issues/677
See: https://docs.fedoraproject.org/en-US/fedora-coreos/alternatives/

 - Remove iptables-legacy starting with Fedora 43
